### DISCUSSION TITLE
Analytics Tracker

### DISCUSSION THREAD

**Discussion by nghiapdx** [vuestorefront/spree#287] [Q&A]
Hello everyone,
How's analytics being integrated into vuestorefront? 
I have Spree and Vue Storefront as back and front end. 

Thank you.

**Comment by rafalcymerys** [comment]
Hi @nghiapdx - you can use the standard nuxt/google-analytics and nuxt-segment-analytics modules:
https://google-analytics.nuxtjs.org/
https://www.npmjs.com/package/@dansmaculotte/nuxt-segment
